client-data/tools/eraser/eraser.js

```diff
@@ -48,7 +48,7 @@
 			var touch = evt.touches[0];
 			target = document.elementFromPoint(touch.clientX, touch.clientY);
 		}
-		if (erasing && target !== Tools.svg) {
+		if (erasing && target !== Tools.svg && target.className.baseVal.indexOf("opcursor") === -1) {
```

Comment:
I don't think the eraser tool should have to be aware of the pointer...
We could have a `<g>` element inside of which all the board elements are, and let the eraser check for that instead.

Reply:
There is a fast way to do this using `Node.contains()` however IE does not support this for svg elements. Do you want to support IE or can I ignore it?